Add note about using timeformat for partition templates (#5465)
influxdata/docs-v2: content/influxdb/cloud-dedicated/admin/custom-partitions/define-custom-partitions.md

_View [partition template part restrictions](/influxdb/cloud-dedicated/admin/custom-partitions/partition-templates/#restrictions)._

{{% note %}}
#### Always provide a time format when using custom partitioning

When defining a custom partition template for your database or table using any
of the `influxctl` `--template-*` flags, always include the `--template-timeformat`

> flag with a time format to use in your partition template. Otherwise time will
> be omitted from the partition template and partitions won't be able to be compacted.

Reviewer: ```suggestion
flag with a time format to use in your partition template.
Otherwise, InfluxDB omits time from the partition template and won't compact partitions.
```
